components/dada-parse/src/parser/code.rs

```diff
@@ -196,9 +213,11 @@ impl CodeParser<'_, '_> {
         let mut expr = self.parse_expr_1()?;
 
         loop {
-            if let Some(expr1) =
-                self.parse_binop(expr, &[Op::DividedBy, Op::Times], Self::parse_expr_1)
-            {
+            if let Some(expr1) = self.parse_binop(
+                expr,
+                &[Op::EqualEqual, Op::LessThan, Op::GreaterThan],
+                Self::parse_expr_1,
+            ) {
```

Comment:
I think the precedence here isn't quite right. I guess the numbering is a bit counterintuitive, now that I think about it, but the idea is that `parse_expr_3` is *lower* precedence than `parse_expr_2`. Maybe we should rename them to count the other way. I think I just wanted to be able to add "new layers" without renumbering anything.

We want to be able to do `x == y * z` and have that parse as `x == (y * z)`, I imagine; for that to work, you want:

* expr 4 => `==`, `<=`, `>=`, etc
* expr 3 => `+`, `-`
* expr 2 => `/`, `*`

Reply:
Ah I just goofed. The numbering is fine. I was even staring at the rust operator precedence table while I wrote it, and still put them in the wrong order.

Reply:
I think it's correct now.